app/business/rankings.py

```diff
@@ -341,14 +350,14 @@ def get_ordered_pb_averages_for_event(event_id):
         filter(UserEventResults.is_blacklisted.isnot(True)).\
         group_by(UserEventResults.id, UserEventResults.user_id, UserEventResults.average, Competition.id, Competition.title, User.username, User.is_verified).\
         order_by(UserEventResults.id.desc()).\
-        values(UserEventResults.user_id, UserEventResults.average, Competition.id, Competition.title, User.username, UserEventResults.comment, User.is_verified)
```

Comment:
So when you remove this line, you're changing the query to just return a list of `UserEventResults` records. The `values(...)` line says to select the specific values you're specifying.

The SQL for this looks something like

```
SELECT <all those fields I specified>
FROM UserEventResults
INNER JOIN User on UserEventResults.user_id = User.id
INNER JOIN CompetitionEvent ...
INNER JOIN Event ...
```

but removing the `value(...)` line basically just only selects the entire `UserEventResults` record and nothing else from the other tables that are joined in the query.